### ISSUE TITLE
Check performance - maybe there is room for improvement?

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/tensorboard_logger#5]

**Comment by meownoid** [comment]
I believe that library should be executed in the separate process since the interprocess communication is mush faster than disk I/O.

**Comment by lopuhin** [comment]
@meownoid yes, this is also a valid approach, thanks for suggestion! Do you have any use-cases where performance is a concern?

**Comment by meownoid** [comment]
@lopuhin learning networks with pytorch for example
